changelogs/fragments/115-add_mysql_full_version_suffix_return_var.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- mysql_info - add mysql_full_version and suffix return variable (https://github.com/ansible-collections/community.mysql/issues/114)
```

```suggestion
- mysql_info - add ``full_version`` and ``suffix`` return variable (https://github.com/ansible-collections/community.mysql/issues/114).
```
to highlight a little bit. we should also put a dot at the end

---

and maybe `variableS`? but i'm not an expert in English:)

---

or maybe even `values`. It's just a suggestion

---

You're right, I changed it, thanks!